Dockerfile

```diff
@@ -30,19 +30,20 @@ RUN chmod +x /usr/bin/zig-wasi
 # Install utilities
 RUN apt-get install -y valgrind
 RUN curl https://wasmtime.dev/install.sh -sSf | bash
+ENV PATH="${HOME}/.wasmtime/bin:${PATH}"
 
 COPY . /app
 WORKDIR /app/deps
 RUN tar -xf wasi-sysroot-16.0.tar.gz
 WORKDIR /app
 
 # Register backends
-RUN update-alternatives --install /usr/bin/gcc gcc $(which gcc-12) 1
-RUN update-alternatives --install /usr/bin/clang clang $(which clang-11) 1
-RUN update-alternatives --install /usr/bin/cc cc $(which gcc) 1
+RUN update-alternatives --install /usr/bin/gcc gcc $(which gcc-12) 5
+RUN update-alternatives --install /usr/bin/cc cc $(which gcc) 4
+RUN update-alternatives --install /usr/bin/clang clang $(which clang-11) 3
 RUN update-alternatives --install /usr/bin/cc cc $(which clang) 2
-RUN update-alternatives --install /usr/bin/cc cc $(which zig) 3
-RUN update-alternatives --install /usr/bin/cc cc $(which zig-wasi) 4
+RUN update-alternatives --install /usr/bin/cc cc $(which zig) 1
+RUN update-alternatives --install /usr/bin/cc cc $(which zig-wasi) 0
```

Comment:
These priorities were set up wildly wrong and needed to be reversed to continue testing in the devcontainer